client/src/pages/ProfileDetails/ProfileDetails.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from 'react';
+import { Theme, createStyles, makeStyles } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
+import { Box, CardMedia } from '@material-ui/core';
+import ProfileDetailCard from '../../components/ProfileDetail/ProfileDetailCard';
+import useStyles from './useStyles';
+import RequestCard from '../../components/ProfileDetail/RequestCard';
+
+export default function ProfileDetails(): JSX.Element {
+  const classes = useStyles();
+
+  //testing data
+  const [profile, setProfile] = useState({
+    coverImg: '',
+    profileImg: '',
+    firstName: 'Norma',
+    lastName: 'Byers',
+    address: 'Toronto, Ontario',
+    description: 'This is testing decription',
+    galleryImg: null,
+    priceRate: 14,
+  });
+
+  //plug in when dashboard data is ready
+  // useEffect(()=>{
+  //   const fetchProfileData = async () => {
+  //     const profileData = await axios.get<any>(`/profile/${userId}`);
+  //     setProfile(profileData);
+  //   }
+  //   fetchProfileData();
+  // },[userId]);
```

Comment:
We don't want to merge in commented code into the main branch. When the time comes where you need to add this functionality, you can fetch in that information. Otherwise, this PR should be in draft while you finish the full implementation.